File: SkyEditor.Core/ConsoleCommands/UI/ViewFiles.vb
Namespace ConsoleCommands.UI
    Public Class ViewFiles
        Inherits ConsoleCommand

        Public Overrides Sub Main(arguments() As String)
            Dim files = CurrentPluginManager.CurrentIOUIManager.OpenFiles
            Console.WriteLine($"{files.Count} files:")
            For count = 0 To files.Count - 1
                Console.WriteLine($"{count}: {files(count).Title}")
            Next
        End Sub
    End Class
End Namespace

File: SkyEditor.Core/ConsoleCommands/Commands/ViewFiles.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SkyEditor.Core.ConsoleCommands.Commands
{
    public class ViewFiles : ConsoleCommand
    {

        protected override void Main(string[] arguments)
        {
            var files = CurrentApplicationViewModel.OpenFiles;
            Console.WriteLine($"{files.Count} files:");
            for (var count = 0; count <= files.Count - 1; count++)
            {
                Console.WriteLine($"{count}: {files[count].Title}");
            }
        }
    }
}
